public/editUser.php

```diff
@@ -85,17 +82,18 @@
             $error_msg = "Password must be at least $CDASH_MINIMUM_PASSWORD_LENGTH characters.";
         }
 
-        $md5pass = md5($passwd);
-        $md5pass = pdo_real_escape_string($md5pass);
+        $password_hash = password_hash($passwd, PASSWORD_DEFAULT);
```

Comment:
Good point.  We should check the return value from `password_hash()` and only call `user::Save()` when it is not `FALSE`.

What should be done in the error case though?
- Silently ignoring the error seems wrong.
- Logging `password_hash returned FALSE` is only slightly more useful.
- Logging the user's (attempted) new password in plaintext feels pretty dirty and insecure.